pkg/cli/create/identity.go

```diff
@@ -58,7 +60,7 @@ func NewCmdCreateIdentity(f genericclioptions.RESTClientGetter, streams generici
 		Example: identityExample,
 		Run: func(cmd *cobra.Command, args []string) {
 			cmdutil.CheckErr(o.Complete(cmd, f, args))
-			cmdutil.CheckErr(o.Run())
+			ocmdhelpers.CustomOIDCDerivedErrorChecker(o.DiscoveryClient, "Identity creation")(o.Run())
```

Comment:
I think, `Identity creation` is passed to point out which command fails. But it is very likely that user knows which command is executed and maybe we can remove this parameter(from also error message). WDYT?